newIDE/app/src/AssetStore/AssetStoreFilterPanel.js

```diff
@@ -0,0 +1,355 @@
+// @flow
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import { t, Trans } from '@lingui/macro';
+import { Accordion, AccordionHeader, AccordionBody } from '../UI/Accordion';
+import Text from '../UI/Text';
+import InlineCheckbox from '../UI/InlineCheckbox';
+import { ColumnStackLayout } from '../UI/Layout';
+import {
+  TagAssetStoreSearchFilter,
+  AnimatedAssetStoreSearchFilter,
+  ObjectTypeAssetStoreSearchFilter,
+  LicenseAssetStoreSearchFilter,
+  DimensionAssetStoreSearchFilter,
+  ColorAssetStoreSearchFilter,
+} from './AssetStoreSearchFilter';
+import { type AssetFiltersState } from './AssetStoreContext';
+import Slider from '@material-ui/core/Slider';
+import FlatButton from '../UI/FlatButton';
+import { Line, Column } from '../UI/Grid';
+import { type RGBColor } from '../Utils/ColorTransformer';
+import { HexColorField } from './HexColorField';
+
+type Choice = {|
+  label: React.Node,
+  value: string,
+|};
+
+type MultipleChoiceFilterProps = {|
+  filterKey: string,
+  title: ?React.Node,
+  choices: Choice[],
+  isChoiceChecked: (choice: string) => boolean,
+  setChoiceChecked: (choice: string, checked: boolean) => void,
+|};
+
+const MultipleChoiceFilter = ({
+  filterKey,
+  title,
+  choices,
+  isChoiceChecked,
+  setChoiceChecked,
+}: MultipleChoiceFilterProps) => {
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Accordion key={filterKey} defaultExpanded>
+          <AccordionHeader>
+            <Text displayInlineAsSpan>{title}</Text>
+          </AccordionHeader>
+          <AccordionBody>
+            <ColumnStackLayout>
```

Comment:
Can we remove margins for this (`noMargin`), or is the AccordionBody not adding margins and so we need it?

Reply:
It looks like AccordionBody not adding margins. Without it, the content is not aligned with the title.

Reply:
Ok 👍